.travis.yml

```diff
@@ -52,3 +52,4 @@ before_script:
   # This simplifies initial set up and provides a very consistent build and test environment.
   - docker pull golang:1.4
 script: ./node_modules/.bin/gulp check:create-cluster
+after_script: npm run codecov
```

Comment:
Hmm... You could just do `./node_modules/.bin/gulp coverage-codecov-upload`. It is consistent with the previous line. What do you think?